### PR TITLE
Hello world, Ranko.py

### PR DESCRIPTION
**Pull request by ranx77**
I think I finally did it, please approve :)

### CODE REVIEW

**Review by IXIena — COMMENTED** [review]
Approved!

**Review by IXIena — APPROVED** [review]
Approved!

File: Hello world, Ranko.py [hunk 1]
```diff
@@ -0,0 +1 @@
+print ("Hello world, Ranko")
```
Comment (halvardssm) [inline Hello world, Ranko.py:1]: please make sure that there always is an empty new line at the end of any file 😉

**Review by halvardssm — APPROVED** [review]
Congrats on your first PR! Only one thing I would recommend to change 💪🏻

**Pull request merged**